verticapy/tests/stats/test_stats.py

```diff
@@ -125,13 +125,23 @@ def test_het_breuschpagan(self, amazon_vd):
         ]
 
     def test_het_goldfeldquandt(self, amazon_vd):
-        result = amazon_vd.groupby(["date"], ["AVG(number) AS number"])
-        result["lag_number"] = "LAG(number) OVER (ORDER BY date)"
-        result = st.het_goldfeldquandt(result, y="number", X=["lag_number"])
+        vdf = amazon_vd.groupby(["date"], ["AVG(number) AS number"])
+        vdf["lag_number"] = "LAG(number) OVER (ORDER BY date)"
+        result = st.het_goldfeldquandt(vdf, y="number", X=["lag_number"])
         assert result["value"] == [
-            pytest.approx(0.0331426182368922),
+            pytest.approx(30.17263128858259),
+            pytest.approx(1.3988910574921388e-55),
```

Comment:
This number is very small. I wonder if it would make the test unstable.

Reply:
it will but we just need a value near to 0.0